test/manual/kerberos.test.js

```diff
@@ -51,7 +51,6 @@ describe('Kerberos', function () {
     });
   });
 
-  // Unskip this test when a proper setup is available - see NODE-3060
```

Comment:
I think either this comment should remain, or the test should be unskipped

Reply:
Comment is back now.